connectivity.py

```diff
@@ -29,30 +30,34 @@ def conn_configure(self, is_connected):
             is_connected (callable): function to invoke to establish initial
                 connectivity status
         """
+        self._conn_status = RunnerStatus.configuring
         with self._connected_lock:
-            self._connected = is_connected()
+            connected = is_connected()
             self.logger.info("Starting in: '{}' state".format(
-                "connected" if self._connected else "disconnected"))
+                "connected" if connected else "disconnected"))
+            self._connected = connected
 
-        if not self._connected:
+        if not connected:
             # per spec, hold the configure method hoping to get connected
             if not self._connected_event.wait(self.timeout().total_seconds()):
                 self._notify_disconnection(RunnerStatus.warning)
                 # start changed to error status notification countdown
                 self._timeout_job = Job(self._notify_disconnection,
                                         self.timeout(), False,
                                         RunnerStatus.error)
+        self._conn_status = RunnerStatus.configured
 
     def conn_stop(self):
+        self._conn_status = RunnerStatus.stopping
```

Comment:
Setting these different status events is basically what the `Runner` class does for us. Can/should we make this connectivity class a `Runner` if we're doing all of this?

Reply:
We can make it a `Runner` by renaming methods to match `configure` and `stop`, and explicitly calling the class from child's `configure` and `stop`, i.e.,
```python
    def stop(self):
        """ Stop the block by closing the underlying publisher """
        PubSubConnectivity.stop(self)
        self._publisher.close()
        super().stop()
```

Reply:
Actually, not needed, the block itself inherits from `Runner` so `status` is available